Modules/MediaPool/classes/class.ilObjMediaPoolGUI.php

```diff
@@ -581,27 +581,27 @@ public function listMedia()
 
         $ilTabs->setTabActive("content");
         $this->setContentSubTabs("content");
-        
+
         if ($ilAccess->checkAccess("write", "", $this->object->getRefId())) {
             $ilToolbar->addButton(
                 $lng->txt("mep_create_mob"),
                 $ilCtrl->getLinkTarget($this, "createMediaObject")
             );
-            
+
             $mset = new ilSetting("mobs");
             if ($mset->get("mep_activate_pages")) {
                 $ilToolbar->addButton(
                     $lng->txt("mep_create_content_snippet"),
                     $ilCtrl->getLinkTarget($this, "createMediaPoolPage")
                 );
             }
-    
+
             $ilToolbar->addButton(
                 $lng->txt("mep_create_folder"),
                 $ilCtrl->getLinkTarget($this, "createFolderForm")
             );
-                        
-            if (trim($mset->get("upload_dir")) != "" && ilMainMenuGUI::_checkAdministrationPermission()) {
+
+            if (trim($mset->get("upload_dir")) != "") {
```

Comment:
This will remove the admin check for this feature.

Please change this to
```(trim($mset->get("upload_dir")) != "" && false)```
instead. I will fix this afterwards.